server/controllers/conversation.js

```diff
@@ -0,0 +1,51 @@
+const asyncHandler = require("express-async-handler");
+const Conversation = require("../models/Conversation");
+const Profile = require("../models/Profile");
+
+exports.createConversationViaProfile = asyncHandler(async (req, res, next) => {
```

Comment:
I think the `createConversation` controller will suffice. Unless there's another reason we are creating a conversation using the `profileId` ?

Also, in our `Conversation` model we are referencing the `User` model under the `members` field, so the logic for this controller does not seem appropriate :)

Reply:
@rajivtitus  This is when dogOwner is looking through the profiles and wants to send message to one of the profiles he can directly do it from here as we are not sharing userId in the list of profiles

Reply:
I just checked the profile controllers and noticed that we are excluding the 'userId' when we return profiles. I guess this controller makes sense in that case 👍🏽